lib/prompt-app-definition.js

```diff
@@ -0,0 +1,58 @@
+const inquirer = require('inquirer');
+const path = require('path');
+
+module.exports = async function promptAppDefinition() {
+  let appDefinitionSettings = await inquirer.prompt([
```

Comment:
`const`

Reply:
while we can use const here, I used let because we're modifying locations property a few lines later.

Reply:
I think we should always use `const` if the reference doesn't change